resources/js/composables/usePaginator.js

```diff
@@ -71,6 +71,10 @@ class Paginator {
     return Math.max(this.meta.value.from - 1, 0);
   }
 
+  setFirst (first) {
+    this.meta.value.from = first;
```

Comment:
```suggestion
    this.meta.value.from = first +1;
```

With the current changes the pagination page is still shown wrong if it contains only one entry per page.

With the suggested changes this problem should be fixed.